kube_proxy/datadog_checks/__init__.py

```diff
@@ -0,0 +1,4 @@
+# (C) Datadog, Inc. 2018-present
```

Comment:
```suggestion
# (C) Datadog, Inc. 2022-present
```

Reply:
To be fair, this file was created in 2018 so it should be 2018 I think (but I know it does not really matters)